tests/js/spec/stores/metricsTagStore.spec.jsx

```diff
@@ -9,17 +9,17 @@ describe('MetricsTagStore', function () {
     it('should add a new tags and trigger the new addition', () => {
       jest.spyOn(MetricsTagStore, 'trigger');
 
-      const tags = MetricsTagStore.getAllTags();
-      expect(tags).toEqual({});
+      const {metricsTags} = MetricsTagStore.getState();
+      expect(metricsTags).toEqual({});
 
-      MetricsTagStore.onLoadTagsSuccess([
+      MetricsTagStore.onLoadSuccess([
```

Comment:
can we have this file in typescript? 🙏

```
act(() => {
        MetricsTagStore.onLoadSuccess([
          {key: 'environment'},
          {key: 'release'},
          {key: 'session.status'},
        ]);
      });
```

if you convert it to ts, you will have to wrap this with act 😉